.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
     "env": {
         "browser": true,
-        "es2020": true,
+        "es2021": true,
```

Comment:
To not have ESlint ignore on spec, add this line of code in line 5 here.

        "jest": true

Reply:
thanks for the tip. less red is better.